docs/content/doc/advanced/external-renderers.en-us.md

```diff
@@ -55,23 +55,23 @@ add one `[markup.XXXXX]` section per external renderer on your custom `app.ini`:
 
 ```ini
 [markup.asciidoc]
-ENABLED = true
+ENABLED         = true
```

Comment:
I thought we no longer do this kind of alignments 😉

Reply:
We dont?

Reply:
At least [app.example.ini](https://github.com/go-gitea/gitea/blob/master/custom/conf/app.example.ini) doesn't do have since a long time now and I generally prefer to not align because it leads to unneccesarly verbose diffs and it's cumbersome sometimes.

Reply:
Ok will delete latest commit ...